WalletWasabi/WabiSabi/Backend/Rounds/RoundParameters.cs

```diff
@@ -85,7 +88,9 @@ public record RoundParameters
 	public int MaxTransactionSize { get; init; } = StandardTransactionPolicy.MaxTransactionSize ?? 100_000;
 	public FeeRate MinRelayTxFee { get; init; } = StandardTransactionPolicy.MinRelayTxFee
 												  ?? new FeeRate(Money.Satoshis(1000));
-	public bool IsTaprootAllowed => AllowedOutputTypes.Contains(ScriptType.Taproot);
+
+	private int MaxVsizeInputOutputPair => AllowedOutputTypes.Max(x => x.EstimateInputVsize() + x.EstimateOutputVsize());
+	private ScriptType MaxVsizeInputOutputPairScriptType => AllowedOutputTypes.MaxBy(x => x.EstimateInputVsize() + x.EstimateOutputVsize());
```

Comment:
Here, you assume wrongly that `AllowedOutputTypes == AllowedInputTypes`.

Reply:
I'm not sure I understand what you mean. If you mean there should be a mix of allowed input types and allowed output types, then I don't think that's the case. It should precisely be the output types, because it's about creating and later on spending an output.

@turbolay you wrote this code, can you help me out?

Reply:
> Allowed input script type doesn't really matter here because it could be used as input side in a standard transaction, not necessarily a remix

Ok, I think you are right.

Reply:
You can resolve the comment.